addon/templates/components/frost-action-bar.hbs

```diff
@@ -11,7 +11,44 @@
     <div class='loading-text'>{{loadingText}}</div>
   {{/if}}
 </div>
-<div class='{{css}}-buttons'>
+<div class='{{css}}-buttons' data-test={{hook 'frost-action-bar-buttons'}}>
```

Comment:
Believe this will throw an console warning (error?) about needing to use html string safe or similar?